kedro_airflow_k8s/config.py

```diff
@@ -132,6 +132,15 @@ def memory(self):
 
 
 class ResourceConfig(Config):
+
+    @property
+    def tolerations(self):
```

Comment:
Add tolerations and labels to DEFAULT_CONFIG_TEMPLATE as it's used by init task.

Reply:
fixed